### DISCUSSION TITLE
Insert image into pdf

### DISCUSSION THREAD

**Discussion by map70** [ImageMagick/ImageMagick#7256] [Help]
### ImageMagick version

7.1.1-30-Q16

### Operating system, version and so on

Widnows 11

### Description

I am looking for a way to insert an specific image into multiple PDF's in the same spot and save them back as pdf.

Any ideas?

Thanks!

**Comment by dlemstra** [comment]
You should probably use something other than ImageMagick. Not sure what you should use but with ImageMagick every page will become an image.
